src/cidr/v4/ipv4_cidr.rs

```diff
@@ -111,6 +109,7 @@ impl Ipv4Cidr {
     }
 
     #[allow(clippy::should_implement_trait)]
+    #[doc(hidden)] // TODO: remove and rely on std FromStr
```

Comment:
Is this function necessary to be hidden?

Reply:
removed this particular change from this PR